xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-ui/src/main/resources/XWiki/Extensions.xml

```diff
@@ -165,39 +166,46 @@
     #end
 
     #if ($indexed)
+      #define ($formContent)
+        &lt;form action="${xwiki.relativeRequestURL}"&gt;
+          #if ($request.section)
+            &lt;input type="hidden" name="section" value="${escapetool.xml($request.section)}" /&gt;
+          #end
+          &lt;input type="hidden" name="search" value="$!{escapetool.xml($request.search)}" /&gt;
+          &lt;input type="hidden" name="recommended" value="$recommended" /&gt;
+          &lt;input type="hidden" name="indexed" value="$indexed" /&gt;
+          &lt;input type="hidden" name="compatible" value="$compatible" /&gt;
+          #if ($indexJobStatus.state != 'RUNNING')
+            &lt;input type="submit" value="${escapetool.xml($services.localization.render('extensions.search.indexed.reindex'))}" name="index_start" class="btn btn-default"/&gt;
+          #end
+        &lt;/form&gt;
+      #end
       #set ($indexJobStatus = $repository.getStatus("wiki:${xcontext.database}"))
       #if ($indexJobStatus)
-        &lt;div class="box infomessage"&gt;
         #if ($indexJobStatus.state != 'FINISHED')
-          $escapetool.xml($services.localization.render('extensions.search.indexed.started', [$xwiki.formatDate($indexJobStatus.startDate)]))
           #set ($discard = $xwiki.jsfx.use('uicomponents/job/job.js'))
           #set ($jobStatusURL = $doc.getURL('get', $escapetool.url({
               'xpage': 'job_status_json',
               'outputSyntax': 'plain',
               'jobId': $indexJobStatus.request.id
             })))
-          &lt;div class="xcontent job-status" data-url="$escapetool.xml($jobStatusURL)"&gt;
-            #displayJobProgressBar($indexJobStatus, true)
-          &lt;/div&gt;
+          #define ($infoFooterContent)
+            $escapetool.xml($services.localization.render('extensions.search.indexed.started', [$xwiki.formatDate($indexJobStatus.startDate)]))
+            &lt;div class="xcontent job-status" data-url="$escapetool.xml($jobStatusURL)"&gt;
+              #displayJobProgressBar($indexJobStatus, true)
+            &lt;/div&gt;
+          #end
         #else
-          $escapetool.xml($services.localization.render('extensions.search.indexed.on', [$xwiki.formatDate($indexJobStatus.startDate)]))
+          #define ($infoFooterContent)
+            $escapetool.xml($services.localization.render('extensions.search.indexed.on', [$xwiki.formatDate($indexJobStatus.startDate)]))
+          #end
         #end
+        #info("$infoFooterContent
+          $formContent")
```

Comment:
hmmm you should really add a comment if the carriage return is on purpose here

Reply:
I just checked it again and it doesn't seem to me like this is needed. I just figured out it wouldn't hurt since it's HTML syntax, and it makes it a bit more readable.